README.md

```diff
@@ -18,7 +18,13 @@ Monitor and get statistics from your Plex server.
 
 ## About
 
-Tautulli is an application that you can run alongside your Plex Media Server to monitor activity and track various statistics. Most importantly, these statistics include what has been watched, who watched it, when and where they watched it, and how it was watched. All statistics are presented in a nice and clean interface with many tables and graphs, which makes it easy to brag about your server to everyone else.
+Tautulli is an application that you can run alongside your Plex Media Server\
```

Comment:
No need for slashes... remove all those... :P

Markdown ignores new line feeds by default...